front/app/api/analysis_insights/useAnalysisInsights.ts

```diff
@@ -7,7 +7,7 @@ import { IInsights, InsightsKeys, IInsightsParams } from './types';
 export const fetchInsights = ({ analysisId }: IInsightsParams) => {
   return fetcher<IInsights>({
     path: `/analyses/${analysisId}/insights`,
-    action: 'get'
+    action: 'get',
```

Comment:
I'm assuming some kind of auto-linter is adding this in, as I din't and it gets added back when I remove it!